opm/input/eclipse/Schedule/Schedule.cpp

```diff
@@ -3015,14 +3015,24 @@ void Schedule::create_first(const time_point& start_time, const std::optional<ti
     sched_state.guide_rate.update( GuideRateConfig() );
     sched_state.rft_config.update( RFTConfig() );
     sched_state.rst_config.update( RSTConfig::first( this->m_static.rst_config ) );
+
+    if (this->m_static.oilVap.has_value()) {
```

Comment:
so the semantics is now that schedule overrides solution for this, since we only ever grab the static one here, right?